---
title: "A Chromosome-level assembly of *Thymus vulgaris*"
author: "Curro Campuzano Jiménez"
date: "2023-06-26"
image: "t_vulgaris.jpg"
---

This semester, I worked with my supervisor [Thomas Bataillon](https://orcid.org/0000-0002-4730-2538) as part of my Master's program at [BiRC](https://birc.au.dk/) on a Bioinformatics project. Our goal was to use **Pacbio HiFi reads to obtain a chromosome-level assembly of Mediterranean thyme**.

![Mediterranean thyme (*T. vulgaris*) Obtained from [Herbari virtual del Mediterrani Occidental](http://herbarivirtual.uib.es/)](t_vulgaris.jpg){fig-align="center" width="40%"}

The primary objective was to use the genome of a closely related species to improve the contiguity of the highly fragmented *de novo* assembly. Although further research is required, we achieved promising results. For example, we increased the N50 from 1.87 Mb (n=133) at the contig level to 48.92 Mb (n=8) at the scaffold level. An outline of the pipeline can be found [here](https://currocam.github.io/BiRC-Thyme/#/section).

Working with plant genomes has been challenging, and I have learned a lot about genome assembly with long reads and plant genomes. Additionally, this project was part of a broader investigation into the genetic and ecological diversity of Mediterranean thyme, a subject that I found fascinating.

I utilized the [GenomeDK](https://genome.au.dk/) cluster at the center for the data analysis pipeline, employing various programs and programming languages, including R, Python, Julia, and Bash.

All the code and the slides I used for the project defense are publicly available on GitHub, which you can access at <https://currocam.github.io/BiRC-Thyme>/
